project/SigarLoader.scala

```diff
@@ -34,7 +34,7 @@ object SigarLoader {
       sigarArtifact := {
         val report = update.value
         val artifactList = report.matching(
-          moduleFilter(organization = sigarLoader.organization, name = sigarLoader.name))
+          moduleFilter(organization = sigarLoader.head.organization, name = sigarLoader.head.name))
```

Comment:
Interesting, is calling `.head` needed now? wondering if it may ever raise an error.

Reply:
Now that `sigarLoader` is a `Seq` with configs for Optional, Provided and Test, we need to pick one of them for the organization and name. I'll create a separate variable for `"io.kamon" % "sigar-loader" % sigarLoaderVersion` so we can get rid of `.head`